tests/test_subgraph/test_tokens.py

```diff
@@ -0,0 +1,35 @@
+from helpers.enums import Network
+from subgraph.queries.tokens import fetch_across_balances
+from tests.test_subgraph.test_data import ACROSS_BALANCES_TEST_DATA
+from copy import deepcopy
+
+def test_fetch_across_balances(mocker):
+    mocker.patch(
+        "subgraph.subgraph_utils.Client.execute",
+        side_effect=[
+            deepcopy(ACROSS_BALANCES_TEST_DATA),
+            {'tokenBalances': []}
+        ],
+    )
+    mocker.patch(
+        "subgraph.queries.tokens.get_across_lp_multiplier",
+        return_value=1.2
+    )
+    expected_bals = {
+        "0x1c1fd689103bbfd701b3b7d41a3807f12814033d": 409.4748806843512,
```

Comment:
Can we maybe construct `expected_bals` as some computation, like:
```python
    expected_bals = {
        "0x1c1fd689103bbfd701b3b7d41a3807f12814033d": some_const * some_multiplier,
```
so that will be more explanatory to people who will read code?